README.md

```diff
@@ -91,18 +91,55 @@ npm test
 ```
 ### Misc
 
-#### MercadoLibre test account creation
-A command line utility to locally create, authorize and store a MeLi test user account has been provided.
+#### MercadoLibre Test Accounts CLI tool
+A command line utility to locally create, authorize and store a MeLi test user account is provided.
+
+__How to use__
+1. Setup env variables in `.env` file: `bin/cli-meli-acocunts/.env`
+
+```
+# Port to use for oauth server. Suggested '3001'
```

Comment:
The application consumes mercadolibre oauth api server. What does this port exactly expose?

Reply:
this port is used for the express server, which listens for the MeLI OAuth callback